.changesets/feat_garypen_router_185_limit_metrics.md

```diff
@@ -0,0 +1,24 @@
+### Add new attributes to the Supergraph Selector Query attribute ([PR #5433](https://github.com/apollographql/router/pull/5433))
+
+Adds support for four new attributes for the supergraph Query instrument:
+ - aliases (the number of aliases in the query)
+ - depth (the depth of the query)
+ - height (the height of the query)
+ - root_fields (the number of root_fields in the query)
+
+You can use this data to understand how your graph is used and to help determine where to set limits, if desired.
```

Comment:
As I understand it `query` is a selector on the supergraph service. I'm a little confused about how (telemetry) attributes come into play here since I thought they were set with the [`attributes` key](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/standard-attributes), which I don't see in the example config.

Is this accurate? @garypen 

```suggestion
### Add new values for the supergraph `query` selector ([PR #5433](https://github.com/apollographql/router/pull/5433))

Adds support for four new values for the supergraph [`query` selector](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/selectors#supergraph):
 - `aliases`: the number of aliases in the query
 - `depth`: the depth of the query
 - `height`: the height of the query
 - `root_fields`: the number of root fields in the query

You can use this data to understand how your graph is used and to help determine where to set limits.
```

Reply:
@Meschreiber I used the wrong terminology due to my unfamiliarity with the feature. I'll accept your suggestion.